tests/acceptance/features/apiSpacesShares/shareSpaces.feature

```diff
@@ -18,6 +18,7 @@ Feature: Share spaces
 
 
   Scenario Outline: A Space Admin can share a space to another user
+    And the administrator has given "Alice" the role "User" using the settings api
```

Comment:
```suggestion
  Scenario Outline: A Space Admin can share a space to another user
    Given the administrator has given "Alice" the role "User" using the settings api
```

Reply:
this step has been added mistakenly. Thanks